branching_xblock/static/html/branching_xblock_edit.html

```diff
@@ -6,7 +6,7 @@
   <div class="errors" style="color:red;"></div>
 </div>
 
-<div class="xblock-actions">
+<div class="xblock-actions" style="position: sticky; bottom: 0; z-index: 10; box-shadow: 0 -2px 6px rgba(0, 0, 0, 0.05); background: #fff;">
```

Comment:
Looks like these are applied from `branching_xblock/static/css/studio_editor.css` now. Removing this still retains the sticky-ness. Is this left over from testing?